crosswalks/swMATH.csv

```diff
@@ -0,0 +1,75 @@
+Property,swMATH
+codeRepository,source_code
+programmingLanguage,programming_languages
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,operating_systems
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,dependencies
+softwareVersion,
+storageRequirements,
+supportingData,
+author,authors
+citation,standard_articles
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,keywords
+license,license_terms
+producer,
+provider,
+publisher,
+sponsor,
+version,
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,description
+identifier,id
+identifier,orms_id
+name,name
+sameAs,homepage
+url,zbmath_url
+relatedLink,related_software
+givenName,
+familyName,
+email,
+affiliation,
+address,
+reviewAspect,
+reviewBody,
+endDate,
+roleName,
+startDate,
+softwareSuggestions,
+maintainer,
+continuousIntegration,
+buildInstructions,
+developmentStatus,
+embargoEndDate,
+funding,
+issueTracker,
+referencePublication,
+readme,
+hasSourceCode,
+isSourceCodeOf,
+collectionSize,articles_count
```

Comment:
I would suggest at this moment to list the added properties that are mapped with schema.org with only the SwMath column and without the schema.org column:
`,articles_count`
`,classification`